metrics/prometheus.go

```diff
@@ -1155,8 +1155,18 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 		values := make([]string, 0, len(rawLabels))
 		labels := make([]string, 0, len(rawLabels))
 		containerLabels := c.containerLabelsFunc(cont)
+		duplicate := false
 		for l := range rawLabels {
-			labels = append(labels, sanitizeLabelName(l))
+			sl := sanitizeLabelName(l)
+			for _, x := range labels {
+			   if sl != x {
+				   duplicate = true
```

Comment:
I think this will stay permanently true, and we will skip all subsequent labels

Reply:
oops, fixing